src/main/groovy/io/github/cosmicsilence/scalafix/ScalafixTask.groovy

```diff
@@ -43,7 +36,6 @@ class ScalafixTask extends SourceTask {
 
     private void processSources() {
         def sourcePaths = source.collect { it.toPath() }
-        def configFile = resolveConfigFile()
```

Comment:
I think this change will break the task. `withConfig` expects a `java.util.Optional<Path>` as argument. We are now passing a `RegularFileProperty`. I think we need to preserve this local def and assign the following to it: `def maybeConfigFile = java.util.Optional.ofNullable(configFile.get()).map { it.asFile.toPath() }`